resources/js/features/forums/components/RecentPostsMainRoot/RecentPostsCards/RecentPostsCards.tsx

```diff
@@ -10,37 +10,38 @@ import { PostTimestamp } from '../PostTimestamp';
 export const RecentPostsCards: FC = () => {
   const { props } = usePage<RecentPostsPageProps>();
 
-  const { recentForumPosts, auth } = props;
+  const { auth, paginatedTopics } = props;
 
   return (
     <div className="flex flex-col gap-y-2">
-      {recentForumPosts.map((recentForumPost) => (
-        <div key={`card-${recentForumPost.commentId}`} className="embedded">
+      {paginatedTopics.items.map((topic) => (
+        <div key={`card-${topic.latestComment.id}`} className="embedded">
           <div className="relative flex justify-between">
             <div className="flex flex-col gap-y-1">
-              <UserAvatar displayName={recentForumPost.authorDisplayName} size={16} />
+              <UserAvatar displayName={topic.latestComment.user.displayName} size={16} />
+
               <span className="smalldate">
                 <PostTimestamp
                   asAbsoluteDate={auth?.user.preferences.prefersAbsoluteDates ?? false}
-                  postedAt={recentForumPost.postedAt}
+                  postedAt={topic.latestComment.createdAt}
                 />
               </span>
             </div>
 
-            <AggregateRecentPostLinks recentForumPost={recentForumPost} />
+            <AggregateRecentPostLinks topic={topic} />
           </div>
 
           <div className="flex flex-col gap-y-2">
             <p className="truncate">
               in{' '}
               <a
-                href={`/viewtopic.php?t=${recentForumPost.forumTopicId}&c=${recentForumPost.commentId}#${recentForumPost.commentId}`}
+                href={`/viewtopic.php?t=${topic.id}&c=${topic.latestComment.id}#${topic.latestComment.id}`}
               >
-                {recentForumPost.forumTopicTitle}
+                {topic.title}
               </a>
             </p>
 
-            <p className="line-clamp-3 text-xs">{recentForumPost.shortMessage}</p>
+            <p className="line-clamp-3 text-xs">{topic.latestComment.body}</p>
```

Comment:
The "N posts in the last 24 hours" / "N posts in the last week" are missing.

master:
![image](https://github.com/user-attachments/assets/39f92c27-ce0c-467b-95dc-3deaaaa9e322)

this branch:
![image](https://github.com/user-attachments/assets/1b5f1726-1f82-4fa0-877b-96ba53a4aba7)

Reply:
Good catch, thanks. When I changed the query around I deleted an `->include()` function call that I forgot to re-add. Fixed in latest.

Reply:
Last 24 hours is showing up now, but still not seeing last 7 days.

master:
![image](https://github.com/user-attachments/assets/0570ad99-d312-4e4c-94de-364ec1bde34e)

this branch:
![image](https://github.com/user-attachments/assets/612a1415-a38b-478a-82f0-4e64f947be77)

Reply:
Looks like there was a logic error in the React code. Fixed in https://github.com/RetroAchievements/RAWeb/pull/2579/commits/8512598cec3d478a4a66bc17ad48302eccc44b1a and added another test case.